setup.py

```diff
@@ -33,7 +33,8 @@
     author_email = "daniel.b.allan@gmail.com",
     url = "https://github.com/soft-matter/trackpy",
     install_requires = ['numpy>=1.8', 'scipy>=0.12', 'six>=1.8',
-	                'pandas>=0.13', 'pyyaml', 'matplotlib'],
+	                'pandas>=0.13', 'pyyaml', 'matplotlib',
+                        'tables'],
```

Comment:
Our previous policy was to *not* require pytables, since pytables is a relatively "heavy" package, and not all trackpy users care about `framewise_data`. (Plus, the vast majority have pytables anyway as part of Anaconda etc.) Is there a reason that we need to require it now?

Reply:
No, I just included it because I thought it was missing, without it you can't use `framewise_data` but if you want to keep it optional, that's fine by me.